R/Analyze_Poisson_PredictBounds.R

```diff
@@ -16,7 +16,7 @@
 #' @param dfTransformed `data.frame` Transformed data for analysis. Data should
 #'   have one record per site with expected columns: `GroupID`, `GroupLevel`,
 #'   `Numerator`, `Denominator`, and `Metric`. For more details see the Data
```

Comment:
```suggestion
#'   `Numerator`, `Denominator`, and `Metric`. For more details see the [Data Model article](https://gilead-biostats.github.io/gsm.core/articles/DataModel.html).
```
I couldn't leave this comment across lines 18 and 19, sadly.